### DISCUSSION TITLE
Feature Request - Elastic Maps Server for Air-Gapped environments

### DISCUSSION THREAD

**Discussion by metapolitics** [Security-Onion-Solutions/securityonion#7594] [Ideas] [answered]
**Feature:** Installation of the Elastic Maps Server within Security Onion Air-Gapped deployments where internet access is not available.

[Elastic offer the ability to self-host their Maps Server](https://www.elastic.co/guide/en/kibana/current/maps-connect-to-ems.html#elastic-maps-server)  where internet access may not be available. I wasn't sure if this was already being covered somewhere - I searched the issues/discussions/projects and release notes but nothing came up.

When deploying SO in Air-Gapped environments, the functionality of mapping geo-locations  in dashboards and reporting is lost. The inclusion of the Maps Server as another docker container alleviates this problem in air-gapped environments, and the docker image contains Zoom Level 8 (country level basemap data). 

I created an associated Issue/Feature request #7593. Wasn't sure on the best place/process to initiate it.

**Comment by dougburks** [comment, marked as answer]
The Elastic Maps Server is 2.3GB and we don't have room for it in our ISO image.

You may be able to manually sneakernet that Docker image and configure it yourself.

**Comment by xfaith** [comment]
Would be interested in this if someone takes it on as an add-on.
